backend/telemetry_core/src/main.rs

```diff
@@ -537,6 +537,10 @@ async fn return_prometheus_metrics(aggregator: AggregatorSet) -> Response<hyper:
             "telemetry_total_messages_to_feeds{{aggregator=\"{}\"}} {} {}\n",
             idx, m.total_messages_to_feeds, m.timestamp_unix_ms
         ));
+        s.push_str(&format!(
+            "telemetry_current_messages_to_aggregator{{aggregator=\"{}\"}} {} {}\n\n",
+            idx, m.current_messages_to_aggregator, m.timestamp_unix_ms
+        ));
```

Comment:
You can change all of these to:

```suggestion
        write!(
            &mut s,
            "telemetry_current_messages_to_aggregator{{aggregator=\"{}\"}} {} {}\n\n",
            idx, m.current_messages_to_aggregator, m.timestamp_unix_ms
        );
```

Reply:
Ooh yeah, that is better, thanks!

Reply:
Done!